src/variorum/Intel/clocks_features.c

```diff
@@ -112,54 +116,62 @@ void dump_clocks_data(FILE *writedest, off_t msr_aperf, off_t msr_mperf, off_t m
         perf_storage(&pd, msr_perf_status);
         if (control_domains == SOCKET)
         {
-            fprintf(writedest, "_CLOCKS_DATA hostname socket aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
+            fprintf(writedest,
+                    "_CLOCKS_DATA hostname socket aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
         }
         else if (control_domains == CORE)
         {
-            fprintf(writedest, "_CLOCKS_DATA hostname socket core thread_phy thread_log aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
+            fprintf(writedest,
+                    "_CLOCKS_DATA hostname socket core thread_phy thread_log aperf mperf tsc curr_freq_mhz avg_freq_mhz\n");
```

Comment:
Looks longer than 80 chars...

Reply:
Astyle counts characters not taking into account the indentation” when enforcing line width. It removes the indentation before calculating the line width. I don't think we can get around this one easily.